ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineGetPayloadBodiesByRangeV1.java

```diff
@@ -64,12 +62,12 @@ public JsonRpcResponse syncResponse(final JsonRpcRequestContext request) {
     final long count = request.getRequiredParameter(1, UnsignedLongParameter.class).getValue();
     final Object reqId = request.getRequest().getId();
 
-    traceLambda(
-        LOG,
-        "{} parameters: start block number {} count {}",
-        () -> getName(),
-        () -> startBlockNumber,
-        () -> count);
+    LOG.atTrace()
+        .setMessage("{} parameters: start block number {} count {}")
+        .addArgument(() -> getName())
+        .addArgument(() -> startBlockNumber)
```

Comment:
Are all the lamdas of variables, parameters, and fields needed?  (follow on, is there a good mechanical way to migrate it?)
```suggestion
        .addArgument(startBlockNumber)
```

Reply:
What I did initially was to implement these in the `Slf4jLambdaHelper`, then inline those with IntelliJ and then unroll the loops. Another option would have been to replace the variable arguments methods with fixed one (I would say there are no more than 5 or 6 var args ever), implement those, and then inline. 
I'll take a look to remove the lambdas when not necessary, thanks for noticing it.